camunda-formio-tasklist-vue/docs/components/TaskList.md

```diff
@@ -17,10 +17,10 @@ solution in your Vue.js based project.
 | formsflowaiApiUrl :triangular_flag_on_post: | string | undefined | The URL of the form-flow API  endpoint.|
 | formsflowaiUrl :triangular_flag_on_post: | string | undefined | The URL endpoint which you would like to host your Vue based application.|
 | containerHeight | string | 100vh | Prop to adjust the height values of Vue component, in case of Vue component is too much for your integrated application. You can adjust the height from the range 100-400 |
-| taskSortBy | string | "created" | Prop to decide by default what value should the tasks be sorted when displayed at first. You can use the values like - "created", "dueDate", "followUpDate", "priority", "name", "assignee" |
-| taskSortOrder  | string | "desc" | Prop to decide by default what value should the tasks should be ordered when displayed at first. You can use the values like - "desc", "asc" |
+| defaultTaskSortBy | string | "created" | Prop to decide by default what value should the tasks be sorted when displayed at first. You can use the values like - "created", "dueDate", "followUpDate", "priority", "name", "assignee" |
+| defaultTaskSortOrder  | string | "desc" | Prop to decide by default what value should the tasks should be ordered when displayed at first. You can use the values like - "desc", "asc" |
 
-🚩- mandatory props to be passed
+:triangular_flag_on_post: - mandatory props to be passed
```

Comment:
what is this for?

Reply:
This is used for showing emojis in github. Using notations like this we can work show emojis in github, you can refer the [emoji cheat sheet mentioned here](https://til.kurianbenoy.com/git-github/using-emojis-in-git-commits)